src/progress/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withTheme } from 'styled-components'
+import Dots from './Dots'
+import Bar from './Bar'
+import Pacman from './Pacman'
+
+export default withTheme(({ index, length, update, theme }) => {
+  const DefaultProgress = (
+    <Dots
+      mx="auto"
+      mb={2}
+      index={index}
+      length={length}
+      onClick={index => {
+        update({ index })
+      }}
+    />
+  )
+
+  switch (theme.progress) {
+    case 'none':
+      return null
+
+    case 'bar':
+      return (
+        <Bar
+          index={index}
+          length={length}
+          onClick={index => {
+            update({ index })
+          }}
+        />
+      )
+
+    case 'dots':
```

Comment:
```jsx
case 'dots':
default:
  return (<Dots ...
```

otherwise you're rendering both dots and bar.

Reply:
You mean I can combine the dots and the default and just return once? 

I didn't know that was valid :/

Reply:
Fixed :)